contrib/packs/actions/workflows/create.yaml

```diff
@@ -0,0 +1,51 @@
+---
+vars:
+  pack_path: '/opt/stackstorm/packs/{{ name }}'
+  pack_group: stanley
+  pack_permissions: '775'
+
+chain:
+  -
+    name: "create directory"
+    ref: "core.local_sudo"
+    parameters:
+      cmd: "mkdir -p {{ pack_path }}"
+    on-success: "change permissions"
+  -
```

Comment:
I think we should init an empty git repo during creation (DZ proposed this as something we should have for v1.0 scope AFAIR), and also have git remote as an optional parameter. We're assuming git all the way, right? And it won't break anything for people who aren't gonna use it, too.